dev/io.openliberty.microprofile.telemetry.2.0.internal/src/io/openliberty/microprofile/telemetry20/internal/rest/TelemetryServletFilter.java

```diff
@@ -142,6 +142,10 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
                 request.setAttribute(SPAN_PARENT_CONTEXT, parentContext);
                 request.setAttribute(SPAN_SCOPE, scope);
 
+                ((HttpServletResponse) response)
+                                .setHeader(ACCESS_TRACE_HEADER_NAME,
+                                           Span.fromContext(spanContext).getSpanContext().getTraceId() + ":" + Span.fromContext(spanContext).getSpanContext().getSpanId());
+
                 if (tc.isDebugEnabled()) {
                     Tr.debug(tc, "Span traceId=" + Span.current().getSpanContext().getTraceId() + ", spanId=" + Span.current().getSpanContext().getSpanId());
```

Comment:
What is the difference between the two approaches of getting the SpanContext?
Did you verify if `Span.current().getSpanContext()` and `Span.fromContext(spanContext).getSpanContext()` `return the same thing?

Reply:
They return the same thing but updated for consistency, thanks.